setup.py

```diff
@@ -74,6 +74,7 @@ def _get_docker_dependency() -> str:
     "pandas<1.4,>=1.3.4",
     "watchdog >=2.1.6",
     "pytest==7.0.0",
+    "web3==5.25.0",
```

Comment:
I prefer we make `open-aea-ledger-ethereum` the dependency. Then we don't need to pay attention between the web3 version here and the one in the plugin.